packages/sdk/app-framework/src/App.tsx

```diff
@@ -49,6 +50,7 @@ export type CreateAppOptions = {
  * @param params.placeholder Placeholder component to render during startup.
  * @param params.fallback Fallback component to render if an error occurs during startup.
  * @param params.cacheEnabled Whether to cache enabled plugins in localStorage.
+ * @param params.safeMode Whether to enable safe mode.
```

Comment:
would be helpful to describe what safe more actually does (e.g. ignores enabled plugins)